code/modules/research/designs/weapon_designs.dm

```diff
@@ -126,7 +126,7 @@
 	category = list("Weapons")
 
 /datum/design/smg
-	name = "Prototype Submachine Gun"
+	name = "NanoTrasen Saber SMG"
```

Comment:
Oh, it's an abstract class? Then why even change it if it's never going to appear anywhere?